app/views/backoffice/dashboard/_completion.en.html.erb

```diff
@@ -1,57 +1,57 @@
-<tr>
-  <td valign="middle" width="200"><%= completion.completed_at %></td>
-  <td valign="middle" width="160"><%= completion.reference_code %></td>
-  <td valign="middle"><%= completion.submission_type || 'N/A' %></td>
-  <td valign="middle"><%= completion.court %></td>
+<tr class="govuk-table__row">
+  <td class="govuk-table__cell"><%= completion.completed_at %></td>
+  <td class="govuk-table__cell"><%= completion.reference_code %></td>
+  <td class="govuk-table__cell"><%= completion.submission_type || 'N/A' %></td>
+  <td class="govuk-table__cell"><%= completion.court %></td>
 </tr>
 
 <% if defined?(@reference_code) %>
   <% c100 = completion.c100_application %>
 
-  <tr>
-    <td valign="middle" colspan="4" class="no-border">
+  <tr class="govuk-table__row">
+    <td colspan="4" class="no-border govuk-table__cell">
       <div class="panel panel-border-narrow application-extra-details">
-        <table>
-          <tr>
-            <td width="300" class="no-border">
-              <dl>
-                <dt>Application started at</dt>
-                <dd><%= completion.started_at %> (v:<%= completion.metadata['v'] || '1-3' %>)</dd>
-                <dt>Saved for later?</dt>
-                <dd><%= completion.metadata['saved_for_later'] %></dd>
-                <dt>Any applicant under 18?</dt>
-                <dd><%= completion.metadata['under_age'] || false %></dd>
-                <dt>Payment type</dt>
-                <dd><%= completion.metadata['payment_type'] || 'N/A' %></dd>
-                <dt>Legal representation</dt>
-                <dd><%= completion.metadata['legal_representation'] || 'N/A' %></dd>
-                <dt>Urgent hearing</dt>
-                <dd><%= completion.metadata['urgent_hearing'] || 'N/A' %></dd>
-                <dt>Without notice hearing</dt>
-                <dd><%= completion.metadata['without_notice'] || 'N/A' %></dd>
-              </dl>
+        <table class="govuk-table">
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell no-border">
+                <dl class="govuk-summary-list govuk-summary-list--no-border">
+                  <dt class="govuk-!-font-size-14">Application started at</dt>
```

Comment:
There is a lot of repetition of classes here. Perhaps better to target the dt and dd from the stylesheet instead?
Like we had before:
https://github.com/ministryofjustice/c100-application/blob/master/app/assets/stylesheets/local/app/backoffice.scss#L13